assets/files.wxs

```diff
@@ -3065,6 +3065,15 @@
       <Component Id="cmpEC42F4F98D70437BB0E290A0E0FE1775" Guid="{5483092a-12c2-4e6c-b730-be491b8909c0}">
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Online help tests for PowerShell Cmdlets.Validate 'get-help Out-GridView -Online'
```powershell
Expected strings to be the same, but they were different.
Expected length: 46
Actual length:   47
Strings differ at index 40.
Expected: '...nkID=11336...'
But was:  '...nkID=21093...'
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\HelpSystem.OnlineHelp.Tests.ps1: line 42
42:                 $actualURI | Should -Be $cmdlet.HelpURI
```